build/bazel/remote/execution/v0.2/remote_execution.proto

```diff
@@ -760,15 +767,16 @@ message ExecuteRequest {
   // omitted.
   string instance_name = 1;
 
-  // The action to be performed.
-  Action action = 2;
-
   // If true, the action will be executed anew even if its result was already
   // present in the cache. If false, the result may be served from the
   // [ActionCache][build.bazel.remote.execution.v0_2.ActionCache].
   bool skip_cache_lookup = 3;
 
-  reserved 4, 5; // Used for removed fields in an earlier version of the API.
+  reserved 2, 4, 5; // Used for removed fields in an earlier version of the API.
```

Comment:
I suggest removing all these from this version. Maybe in a different patch.

Reply:
I think it would be good to do in in a different patch, yes. If we're comfortable saying that these are Completely Different Protos and should never be interchanged, then I think it's fine to renumber fields or at least reuse numbers.. I am comfortable saying that personally but it should probably be done when all the dust is settled anyway.